pkg/sources/github/github.go

```diff
@@ -674,6 +677,7 @@ func (s *Source) addReposByApp(ctx context.Context, apiClient *github.Client) er
 }
 
 func (s *Source) addAllVisibleOrgs(ctx context.Context, apiClient *github.Client) {
+	log.Debug("enumerating all visibile organizations on GHE")
```

Comment:
__optional__: Unless it unnecessarily complicates the log message, might be best to use `s.log` for all of these `log.Debug`s that were added.